target/linux/ath79/mikrotik/config-default

```diff
@@ -1,3 +1,6 @@
+# CONFIG_AG71XX_LEGACY is not set
+# CONFIG_AR8216_PHY is not set
+# CONFIG_AT803X_PHY is not set
```

Comment:
Please do this in a separated PR so that this PR can be accepted as soon as possible.